docs/docsite/rst/contributing/importing.rst

```diff
@@ -238,6 +238,12 @@ rather than the repository name.
     to lowercase, and translating '-'  and '.' to '_'. If the name of an existing role should not be
     altered, don't set the value of *role_name*.
 
+Creating Organizational Namespaces
+==================================
+
+At the moment organization namespaces have to be added manually. To request a new namespace for your GitHub org
```

Comment:
This should read something more like: 

In order to protect against copyright and trademark infringements, new Galaxy namespaces can only be created by submitting a request to the Galaxy team. A team member will review the request and create the new namespace within 1 to 2 business days.

When submitting a request, please include a link to the GitHub organization and a list of Galaxy usernames to be given ownership rights to the new namespace. `Click here to submit a request now <https://github.com/ansible/galaxy/issues/new?template=New_namespace.md>`_.